api/src/index.node.ts

```diff
@@ -83,11 +82,16 @@ startRouteProbeWatcher(watchDir);
 // Set up websocket server
 setupRealtimeService({ server, path: "/ws", wsConnections });
 
+// set up webhonc manager
+webhonc.setupWebhonc({ host: resolveWebhoncUrl(), db, wsConnections });
+
 // check settings if proxy requests is enabled
-const proxyRequestsEnabled = await getSetting(db, "proxyRequestsEnabled");
+const proxyRequestsEnabled = (await getSetting(
+  db,
+  "proxyRequestsEnabled",
+)) as boolean;
```

Comment:
can avoid the ts coercion and just make this value a bool with `!!`

```suggestion
const proxyRequestsEnabled = !!(await getSetting(
  db,
  "proxyRequestsEnabled",
));
```